src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.ts

```diff
@@ -420,6 +423,7 @@ export class BasicAutocompleteComponent<O, V = O>
       asValue: this.valueMapper(opt),
       asString: this.optionToString(opt),
       selected: false,
+      isHidden: (opt as any)?.isHidden ?? false,
```

Comment:
would this also work?

```suggestion
      isHidden: (opt as SelectableOption)?.isHidden ?? false,
```